packages/fxa-content-server/app/scripts/views/mixins/external-links-mixin.js

```diff
@@ -37,7 +36,7 @@ export default {
 
     event.preventDefault();
 
-    return this._flushMetricsThenRedirect(event.currentTarget.href);
+    this.navigateAway(event.currentTarget.href);
```

Comment:
This is just a simplification, `_flushMetricsThenRedirect` had the same effect as `navigateAway`, no reason to have two methods that do the same thing.